src/legacy/core_plugins/kibana/server/routes/api/export/index.js

```diff
@@ -47,7 +47,7 @@ export function exportApi(server) {
             .header('Content-Type', 'application/json')
             .header('Content-Length', Buffer.byteLength(json, 'utf8'));
         })
-        .catch(err => Boom.boomify(err, { statusCode: 400 }));
+        .catch(err => Boom.boomify(err, { statusCode: 500, override: false }));
```

Comment:
I suspect we used to return a 400 error code assuming any exception from the import was caused by https://github.com/elastic/kibana/commit/68bdd7cbdd81d3ecf39bdaa0975d40fa5aa4d04b#diff-4f5b0a776f963bfbb642b07ab2152b42L16

Reply:
I'm not entirely sure, but as I understand it is default `hapi` behaviour and we can remove this wrapper.
> Any error thrown by a lifecycle method is converted into a boom object and defaults to status code 500 if the error is not already a boom object.

https://hapijs.com/api/17.9.0#errors